neurokit2/ppg/ppg_clean.py

```diff
@@ -33,27 +35,62 @@ def ppg_clean(ppg_signal, sampling_rate=1000, method="elgendi"):
     >>> import matplotlib.pyplot as plt
     >>>
     >>> ppg = nk.ppg_simulate(heart_rate=75, duration=30)
-    >>> ppg_clean = nk.ppg_clean(ppg)
+    >>> ppg_elgendi = nk.ppg_clean(ppg, method='elgendi')
+    >>> ppg_nabian = nk.ppg_clean(ppg, method='nabian2018', heart_rate=75)
     >>>
-    >>> plt.plot(ppg, label="raw PPG") #doctest: +SKIP
-    >>> plt.plot(ppg_clean, label="clean PPG") #doctest: +SKIP
-    >>> plt.legend() #doctest: +SKIP
+    >>> # Plot to compare methods
+    >>> signals = pd.DataFrame({"PPG_Raw" : ppg,
+    ...                         "PPG_Elgendi" : ppg_elgendi,
+    ...                         "PPG_Nabian" : ppg_nabian})
+     >>> signals.plot() #doctest: +ELLIPSIS
+     <matplotlib.axes._subplots.AxesSubplot object at ...>
+
+    References
+    ----------
+    - Nabian, M., Yin, Y., Wormwood, J., Quigley, K. S., Barrett, L. F., &amp; Ostadabbas, S. (2018).
+    An Open-Source Feature Extraction Tool for the Analysis of Peripheral Physiological Data. IEEE Journal of
+    Translational Engineering in Health and Medicine, 6, 1-11. doi:10.1109/jtehm.2018.2878000
 
     """
     ppg_signal = as_vector(ppg_signal)
 
     method = method.lower()
     if method in ["elgendi"]:
         clean = _ppg_clean_elgendi(ppg_signal, sampling_rate)
+    elif method in ["nabian2018"]:
+        clean = _ppg_clean_nabian2018(ppg_signal, sampling_rate, heart_rate=heart_rate)
     else:
-        raise ValueError("Neurokit error: Please use one of the following methods: 'elgendi'.")
+        raise ValueError("Neurokit error: Please use one of the following methods: 'elgendi' or 'nabian2018'.")
 
     return clean
 
+# =============================================================================
+# Methods
+# =============================================================================
 
 def _ppg_clean_elgendi(ppg_signal, sampling_rate):
 
     filtered = signal_filter(
         ppg_signal, sampling_rate=sampling_rate, lowcut=0.5, highcut=8, order=3, method="butter_ba"
     )
     return filtered
+
+
+def _ppg_clean_nabian2018(ppg_signal, sampling_rate, heart_rate=None):
```

Comment:
@elaineteo2000 Great attempt on your first PR Elaine! 😄 

I've reformatted the function to have:
1) ``highcut`` not as an argument since the general API of our _clean* functions do not require users to specify any filter related info, so we will just set highcut=40 in the function itself for now

2) Initially you specified that if highcut is not default (40Hz in this case), then it would be at least 10 times the heart rate and less than 0.5times the sampling rate. However ``highcut`` needs a specific value and not a range which is what I think you were trying to do? 😄 So what I've done here is just to check that the default value of 40Hz is meets these above criteria since ``highcut=40`` is the unchangeable, default cut off for now!

3) Great use of ``nk.signal_filter()``! 👍 I put ``highcut=highcut`` here because I have saved (line 84) ``highcut=40`` , initially I think you put ``highcut='default'`` and remember the ``signal_filter()`` function won't be able to process this argument because it is a string and not a number!